src/main/resources/static/cas1-schemas.yml

```diff
@@ -375,6 +375,8 @@ components:
           type: string
         keyWorkerAllocation:
           $ref: '#/components/schemas/Cas1KeyWorkerAllocation'
+        status:
+          $ref: '#/components/schemas/Cas1SpaceBookingSummaryStatus'
```

Comment:
as this is a new field not a breaking change for the API in my mind. That said, I know we do some magic in the UI tht auto-generates the typescript types -will this potentially cause issues on the FE side?